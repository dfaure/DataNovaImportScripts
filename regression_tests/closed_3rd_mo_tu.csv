10364A;HERNY AP;2021-02-23;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-30;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-01;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-21;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-02;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-22;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-01;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-21;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-23;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-04;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-25;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-24;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-01;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-03;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-16;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-14;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-17;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-18;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-17;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-12;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-09;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-16;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-15;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-13;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-11;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-19;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-17;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-10;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-19;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-15;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-11;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-11;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-25;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-09;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-24;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-02;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-10;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-06;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-07;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-01-25;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-07;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-06;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-08;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-04;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-05;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-20;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-05;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-12;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-08;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-13;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-04;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-12;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-16;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-01-27;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-14;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-29;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-20;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-26;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-05;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-27;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-01-29;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-03;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-03-18;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-22;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-31;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-26;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-03-28;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-22;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-27;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-28;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-02-02;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-01-26;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-01-28;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-02-18;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-13;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-15;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-01-30;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-06;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-01-23;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-03;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-08;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-19;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-20;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-09;10:00-12:00;;;;;;;11:45;11:45;11:45;;FERME;;;;;;;
10364A;HERNY AP;2021-04-21;FERME;;;;;;;;;;;FERME;;;;;;;
10364A;HERNY AP;2021-04-10;09:00-11:00;;;;;;;11:00;11:00;;;FERME;;;;;;;
