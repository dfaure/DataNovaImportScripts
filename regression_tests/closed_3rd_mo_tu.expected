10364A|HERNY AP|Mo-Tu,Th-Fr 10:00-12:00; Mo[3],Tu[3] off; Sa 09:00-11:00; PH off; 2021 Apr 02 off
